File: combo.js
var fs = require('fs'),
		translation1 = require('./nj_new_jersey_zip_codes_geo.min.json'),
		translation2 = require('./pa_pennsylvania_zip_codes_geo.min.json');

(function () {
	var key, big, small, res = {};
	if (Object.keys(translation1).length >= Object.keys(translation2).length) {
		big = translation1;
		small = translation2;
	}
	else {
		big = translation2;
		small = translation1;
	}
	for (key in big) {
		if (big.hasOwnProperty(key)) {
			res[key] = (small[key] !== void 0) ? small[key] : big[key];
		}
	}
	fs.writeFileSync('res.json', JSON.stringify(res));
})();

File: combo.py
import json

with open("nj_new_jersey_zip_codes_geo.min.json") as fo:
    data1 = json.load(fo)

with open("pa_pennsylvania_zip_codes_geo.min.json") as fo:
    data2 = json.load(fo)

data1.update(data2)

with open("merge.json", "w") as fo:
    json.dump(data1, fo)